code-studio/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@cdo/code-studio",
+  "version": "0.0.1",
+  "description": "Pipeline for static assets used by our Code Studio rails app (i.e. dashboard)",
+  "author": "Code.org",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=0.10.25 <0.13",
+    "npm": "^2.9.1"
+  },
+  "scripts": {
+    "build": "browserify --debug src/js/blockly_editor.js -o built/js/blockly_editor.js -t [ babelify --presets [es2015] ]",
+    "build:min": "browserify src/js/blockly_editor.js -t [ babelify --presets [es2015] ] | uglifyjs -o built/js/blockly_editor.js",
+    "prebuild": "mkdir -p built/js",
+    "prebuild:min": "mkdir -p built/js",
+    "clean": "rm -rf built",
```

Comment:
I've renamed directory `built` => `build` to be consistent with other places in our repo.